.eslintrc.json

```diff
@@ -0,0 +1,23 @@
+{
+    "env": {
```

Comment:
We already have an ESLint configuration and this might cause conflicts.
Our current configuration does take precedence over this and so this will get ignored, but you need to remove this.
The environment configuration is improper as well.